curriculum/challenges/english/03-front-end-libraries/jquery/target-html-elements-with-selectors-using-jquery.english.md

```diff
@@ -32,8 +32,7 @@ tests:
   - text: You should only use jQuery to add these classes to the element.
     testString: 'assert(!code.match(/class.*animated/g));'
   - text: Your jQuery code should be within the <code>$(document).ready();</code> function.
-    testString: assert(code.match(/\$\(document\)\.ready\(function.*(\s|\n)*.*button.*.addClass.*\);/g));
-
+    testString: assert(code.match(/\$\(document\)\.ready\(function\s*\(\)\s*{\n\s*\$.*button.*.addClass.*/g));
```

Comment:
```suggestion
    testString: 'assert($("button").hasClass("animated") && $("button").hasClass("bounce"));'
```

I vote to just use the same test as the first test. Like @Sky020 mentioned - there's ways around it, but it pretty closely ensures that their code is in the function. And I like keeping the test text there as a reminder for users to put their code in it.